src/DynamoCoreWpf/ViewModels/Core/GalleryViewModel.cs

```diff
@@ -104,6 +101,30 @@ public GalleryViewModel(DynamoViewModel dynamoViewModel)
         }
 
         /// <summary>
+        /// Set image path relative to the gallery Directory.
+        /// It looks for the images side-by-side with the gallerycontents.xml,
+        /// if it's can't be found, it looks one level up in gallery/Data
+        /// </summary>
+        /// <param name="galleryDirectory">
+        /// The directory in which the gallerycontents.xml resides
+        /// </param>
```

Comment:
Perhaps this would have been better:

```csharp
/// <summary>
/// Set image path relative to the gallery directory. The method looks for the 
/// images alongside GalleryContents.xml. If a given image cannot be found, it 
/// searches under "Data" directory in the parent directory.
/// </summary>
```

Does that make sense?